stdlib/io.pyi

```diff
@@ -105,6 +105,7 @@ class FileIO(RawIOBase, BinaryIO):  # type: ignore[misc]  # incompatible definit
     def closefd(self) -> bool: ...
     def write(self, b: ReadableBuffer, /) -> int: ...
     def read(self, size: int = -1, /) -> bytes: ...
+    def readinto(self, buffer: WriteableBuffer, /) -> int: ...
```

Comment:
As @AlexWaygood correctly pointed out, this can also return None (https://github.com/python/cpython/blob/2a5d1eb7073179a13159bce937afdbe240432e7d/Modules/_io/fileio.c#L678).